facturacion_mexico/one_offs/backfill_folio_fiscal.py

```
@@ -0,0 +1,103 @@
+Reporta: revisadas, actualizadas, limpiadas, sin_cambio, errores.
+"""
+
+import frappe
+
+from facturacion_mexico.facturacion_fiscal.utils import (
+	_FOLIO_VIGENTE_STATES,
+	sincronizar_folio_fiscal,
+)
+
+
+def _folio_vigente_esperado(si_name):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_folio_vigente_esperado`

(ANN202)